bindings/python/tests/test_pydantic.py

```diff
@@ -0,0 +1,77 @@
+from datetime import datetime
+from typing import List, Optional
+
+import dlite
```

Comment:
Not  accessed

Reply:
Well spotted. Removed

Reply:
... readded again because it is needed after I improved the tests.